### PR TITLE
Fix Dapr microservices sample

### PR DESCRIPTION
**Pull request by ASHIQUEMD**
### This PR fixes below Dapr microservices sample issue

On running [`rad run dapr.bicep`](https://docs.radapp.io/tutorials/dapr/#step-4-deploy-and-run-the-frontend-application), it throws below error.

`No application was specified. Use --application to specify the application name.`

### CODE REVIEW

**Comment by AaronCrawfis** [comment]
Thanks @ASHIQUEMD! The .rad folder is a great addition to the samples repo to be able to quickly clone and run.

@willtsai could you take a look and make sure this lines up with the sample and the instructions? I noticed we have the application defined in the dapr.bicep file. We can probably remove that from the file and move to the .rad folder instead to match the convention we've been following in the getting started guide and new app tutorial. We also should make sure the app name is consistent throughout the docs as well.

File: samples/dapr/.rad/rad.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+workspace:
+  application: "dapr-app"
```
Comment (willtsai) [inline samples/dapr/.rad/rad.yaml]: The application name in the bicep definition is just `dapr`

```suggestion
  application: "dapr"
```

**Review by willtsai — CHANGES_REQUESTED** [review]
Thank you @ASHIQUEMD for the contribution! I believe the application name in `.rad` must match the application name defined in the Bicep files, so I made a suggestion. please let me know if it makes sense.

@AaronCrawfis - I will create a separate PR to edit the [Dapr Tutorial](https://docs.radapp.io/tutorials/dapr/) in the Documentation to leverage `.rad` for application name.

Let me know if you have additional questions.

**Comment by willtsai** [comment]
corresponding changes to the tutorial in the documentation: https://github.com/radius-project/docs/pull/913

**Comment by willtsai** [comment]
@ASHIQUEMD -- two things:
1. Can you please update your commit message to include a signoff so that this PR may fulfill the [DCO](https://github.com/radius-project/samples/pull/654/checks?check_run_id=18196293890) requirements?
2. Please take a look at my minor suggestion and commit it in if you agree.

Thanks again for your contribution!

**Comment by willtsai** [comment]
@ASHIQUEMD -- can you please take a look at my suggestions and provide your DCO signoff on this PR so that we may get it committed and merged? Thank you!

**Review by willtsai — APPROVED** [review]

**Pull request merged**